project/SwaggerModelGenerator.scala

```diff
@@ -42,7 +45,7 @@ object SwaggerModelGenerator extends AutoPlugin {
       "io.k8s.kubernetes.pkg.apis.batch.v1",
       "io.k8s.kubernetes.pkg.apis.networking.v1"
     )
-    allowedPrefixes.exists(className.startsWith)
+    allowedPrefixes.exists(className.startsWith) && !skipClases.contains(className)
```

Comment:
Typo in `skipClases`, should be `skipClasses`?

Reply:
Fixed